seacrowd/sea_datasets/asr_indocsc/asr_indocsc.py

```diff
@@ -0,0 +1,191 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+# no bibtex citation
+_CITATION = ""
+_DATASETNAME = "asr_indocsc"
+_DESCRIPTION = """\
+This open-source dataset consists of 4.54 hours of transcribed Indonesian
+conversational speech on certain topics, where seven conversations between two
+pairs of speakers were contained.
```

Comment:
The rest looks good to me! Just need to add a description that you have to sign up and/or login first before running this. Thanks! 😃

Reply:
Thanks for checking. I've added the instructions and pushed the change.